perf.groovy

```diff
@@ -44,7 +44,7 @@ def generate(boolean isPr) {
         prTrigger.permitOrg('Microsoft')
         prTrigger.permitOrg('dotnet')
         prTrigger.setCustomTriggerPhrase("(?i)^\\s*(@dotnet-bot\\s+)?(re)?test\\s+perf(\\s+please)?\\s*\$" )
-        prTrigger.triggerForBranch('master');
```

Comment:
All changed in this file get reverted except for this.  Here you pass branchName instead (variable from https://github.com/dotnet/roslyn/pull/17327/files#diff-bc9a87b2e014aa8266aab4a55fbbc50cL10)